topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -23,32 +23,32 @@ This practical aims to familiarize you with the Galaxy user interface. It will t
 Suppose you get the following question:
 
 > ### {% icon question %} Question
-> *Mom (or Dad) ... Which coding exon has the highest number of single nucleotide polymorphisms (SNPs) on human chromosome 22?*
+> *Which coding exon has the highest number of single nucleotide polymorphisms (SNPs) on human chromosome 22?*
 {: .question}
 
 You are thinking "Wow! This is a simple question... I know where to find the data, at the [UCSC Genome Browser](https://genome.ucsc.edu/), but how do I actually compute this?" There is really a straightforward way of answering this question and it is called **Galaxy**. So let's try it...
 
-Browse to your Galaxy instance and log in or register. The Galaxy interface consists of three main parts. The available tools are listed on the left, your analysis history is recorded on the right, and the middle pane will show the home page, the selected tool or the dataset.
+Browse to your Galaxy instance and log in or register. The Galaxy interface consists of three main parts. The available tools are listed on the left, your analysis history is recorded on the right, and the middle pane will show the home page, a tool form or some dataset content.
 
 ![](../../images/galaxy_interface.png)
```

Comment:
Can you add an `alt` text for the images, if possible? And do you know that you can add a legend for the images by adding it right after the path to the image (in the curly brackets)?

Reply:
> And do you know that you can add a legend for the images by adding it right after the path to the image (in the curly brackets)?

You suppose you mean the round brackets.

Reply:
Yes, inside parentheses